src/Symfony/Component/Security/Core/SecurityContext.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Security\Core;
 
+trigger_error('The '.__NAMESPACE__.'\SecurityContext class is deprecated since version 2.6 and will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
There is already a `trigger_error` in the methods, e.g. `setToken`. It does not make sense to have 2 deprecation warnings at the same time. So the ones in the methods can be removed. Also please add here that people should use `Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage` or `Symfony\Component\Security\Core\Authorization\AuthorizationChecker` instead